int32 emg
float64 volt
float64 cur
int32 foot_operation
int32[] foot_x_cordinate
int32 foot_mode
int32 foot_motor_l
int32 foot_motor_r
int32 arm_retorg_req
int32 arm_drill_req
int32[] arm_drill_width
int32 arm_ridge_width
int32 arm_ridge_height
bool database_inforeq
int8 database_vegetable_num1
int8 database_vegetable_num2
int32 database_ridge_width
int32 database_ridge_length
bool webapp_loginpermit
string[] webapp_mainseedlist
string[] webapp_subseedlist
int32 webapp_batterycharge
int32[] webapp_seed_type
int32[] webapp_coordinates_x
int32[] webapp_coordinates_y
int32 webapp_mileage
int32 webapp_completion
bool webapp_operatingstate
